### ISSUE TITLE
The extension in chrome causes HBO Now to throw HDCP errors.

### ISSUE THREAD

**Issue by crkinard** [EFForg/privacybadger#2026]
As the title says. If privacy badger is loaded trying to watch videos on HBO Now causes the video to throw HDCP errors.

Only completely disabling the extension and restarting chrome fixes the issue.

**Comment by ghostwords** [comment]
Counts of error reports that mention HDCP grouped by date and site domain:
```
+---------+-----------------+-------+
| ym      | fqdn            | count |
+---------+-----------------+-------+
| 2018-05 | play.hbogo.com  |    14 |
| 2018-04 | play.hbonow.com |     9 |
| 2018-03 | play.hbogo.com  |    12 |
| 2018-03 | play.hbonow.com |     8 |
| 2018-02 | play.hbonow.com |    11 |
| 2017-12 | play.hbonow.com |     5 |
| 2017-11 | play.hbogo.com  |     2 |
| 2017-11 | play.hbonow.com |     1 |
| 2017-10 | play.hbogo.com  |     3 |
...
```

Counts of error reports that mention HDCP grouped by page domain and exact blocked subdomain:
```
+-----------------+-----------------------------+-------+
| fqdn            | blocked_fqdn                | count |
+-----------------+-----------------------------+-------+
| play.hbogo.com  | individualization.adobe.com |     7 |
| play.hbonow.com | 6356875.fls.doubleclick.net |     5 |
| play.hbogo.com  | assets.adobedtm.com         |     5 |
| play.hbogo.com  | tags.bluekai.com            |     5 |
| play.hbonow.com | googleads.g.doubleclick.net |     4 |
| play.hbogo.com  | hbo.tt.omtrdc.net           |     4 |
| play.hbonow.com | individualization.adobe.com |     4 |
| play.hbonow.com | tags.bluekai.com            |     4 |
| play.hbonow.com | www.googleadservices.com    |     4 |
| play.hbogo.com  | www.googletagmanager.com    |     4 |
| play.hbonow.com | 8019195.fls.doubleclick.net |     3 |
| play.hbogo.com  | cm.everesttech.net          |     3 |
| play.hbogo.com  | dpm.demdex.net              |     3 |
| play.hbonow.com | hbo.tt.omtrdc.net           |     3 |
| play.hbogo.com  | hbolag.demdex.net           |     3 |
| play.hbonow.com | assets.adobedtm.com         |     2 |
| play.hbonow.com | connect.facebook.net        |     2 |
| play.hbonow.com | dpm.demdex.net              |     2 |
| play.hbonow.com | www.googletagmanager.com    |     2 |
| play.hbogo.com  | cdn.krxd.net                |     1 |
| play.hbonow.com | cm.everesttech.net          |     1 |
| play.hbogo.com  | cws.conviva.com             |     1 |
| play.hbonow.com | hbolag.demdex.net           |     1 |
+-----------------+-----------------------------+-------+
```

**Comment by ghostwords** [comment]
It looks like we have a problem where Badgers learn to block HBO domains (`api.ugw.hbogo.eu`, `hls3.pro11.lv3.cdn.hbogo.com`, `comet.api.hbo.com`) on HBO sites (`hbogo.cz`, `play.hbonow.com`, `play.hbogo.com`). May be another issue on top of the HDCP problem.

**Comment by crkinard** [comment]
In the process to get it working I dumped my browser cache. Even my DNS cache. That didn't work. Then I removed the extension and it worked. 

I just re-added it to try and disable it only on HBO Now and without disabling it on HBO Now... it now works. >.<

**Comment by ghostwords** [comment]
This suggests Privacy Badger learns to block some domain required for video playback. So it'll probably happen again to you. But then disabling Privacy Badger just on the HBO site should help (although you may need to restart your browser).

**Comment by crkinard** [comment]
👍  I watch on my PS4 at home mostly. Just had some time during lunch to watch John Oliver.

**Comment by ghostwords** [comment]
I have a feeling that `individualization.adobe.com` is required for HDCP. Maybe other domains too, but probably this one.

Could somebody confirm sometime blocking `individualization.adobe.com` on HBO causes HDCP errors and unblocking restores access? Does "cookieblocking" (moving the slider to yellow and reloading the page) the domain allow HBO to function? I'm guessing it does not.

**Comment by ghostwords** [comment]
Yep, based on user error reports, blocking `individualization.adobe.com` is responsible for playback errors on lots of sites:
```
+----------------------------+-----------------------------+-------+
| fqdn                       | blocked_fqdn                | count |
+----------------------------+-----------------------------+-------+
| go.cnn.com                 | individualization.adobe.com |    17 |
| www.hulu.com               | individualization.adobe.com |    17 |
| play.hbogo.com             | individualization.adobe.com |    12 |
| tv.xfinity.com             | individualization.adobe.com |     8 |
| play.hbonow.com            | individualization.adobe.com |     7 |
| www.channel4.com           | individualization.adobe.com |     7 |
| www.hbonow.com             | individualization.adobe.com |     5 |
| xfinitytv.comcast.net      | individualization.adobe.com |     4 |
| se.hbonordic.com           | individualization.adobe.com |     3 |
| www.cartoonnetwork.com     | individualization.adobe.com |     3 |
| teamcoco.com               | individualization.adobe.com |     2 |
| www.cnn.com                | individualization.adobe.com |     2 |
| www.fox.com                | individualization.adobe.com |     2 |
| www.trutv.com              | individualization.adobe.com |     2 |
| www.tvnow.de               | individualization.adobe.com |     2 |
| dk.hbonordic.com           | individualization.adobe.com |     1 |
| espn.go.com                | individualization.adobe.com |     1 |
| fi.hbonordic.com           | individualization.adobe.com |     1 |
| no.hbonordic.com           | individualization.adobe.com |     1 |
| player.bfi.org.uk          | individualization.adobe.com |     1 |
| time.com                   | individualization.adobe.com |     1 |
| video2.timewarnercable.com | individualization.adobe.com |     1 |
| www.adultswim.com          | individualization.adobe.com |     1 |
| www.clipfish.de            | individualization.adobe.com |     1 |
| www.globaltv.com           | individualization.adobe.com |     1 |
| www.n-tv.de                | individualization.adobe.com |     1 |
| www.nowtv.de               | individualization.adobe.com |     1 |
| www.syfy.com               | individualization.adobe.com |     1 |
...
```

**Comment by andresbase** [comment]
When I tried it on Chrome/macOS the domain I had to move to yellow was `hbo.tt.omtrdc.net`
I would not be able to even log in without that. After moving to cookie block the mentioned domain everything worked including playing videos.

```
**** ACTION_MAP for omtrdc.net
VM193:5 adobe.tt.omtrdc.net {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1525599919028,
  "userAction": ""
}
VM193:5 amazoncustomerservice.d2.sc.omtrdc.net {
  "dnt": false,
  "heuristicAction": "cookieblock",
  "nextUpdateTime": 0,
  "userAction": ""
}
VM193:5 amazonwebservices.d2.sc.omtrdc.net {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1522866708836,
  "userAction": ""
}
VM193:5 amcglobal.sc.omtrdc.net {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1525646350327,
  "userAction": ""
}
VM193:5 attservicesinc.tt.omtrdc.net {
  "dnt": false,
  "heuristicAction": "cookieblock",
  "nextUpdateTime": 0,
  "userAction": ""
}
VM193:5 omtrdc.net {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 0,
  "userAction": ""
}
VM193:5 hbo.tt.omtrdc.net {
  "userAction": "user_allow",
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1526989454169
}
VM193:7 **** SNITCH_MAP for omtrdc.net
VM193:9 omtrdc.net [
  "amazon.com",
  "myportfolio.com",
  "adobe.com"
]
```

**Comment by crkinard** [comment]
@andresbase

I think it is this too. Used UBlock Origin to block "omtrdc.net" and I got the HDCP errors. Unblocked and back to normal (working).

**Comment by ghostwords** [comment]
Something must have changed because we are now getting fewer reports from HBO sites:
```
+---------+-------+
| ym      | count |
+---------+-------+
| 2019-02 |     2 |
| 2019-01 |     5 |
| 2018-12 |     2 |
| 2018-11 |    12 |
| 2018-10 |    11 |
| 2018-09 |     9 |
| 2018-08 |    11 |
| 2018-07 |    10 |
| 2018-06 |    12 |
| 2018-05 |    12 |
| 2018-04 |    12 |
| 2018-03 |     6 |
| 2018-02 |     5 |
| 2018-01 |     5 |
...
```
